kotlin-reference-server/src/main/kotlin/org/stellar/reference/dao/TransactionKYCRepository.kt

```diff
@@ -18,12 +18,17 @@ interface TransactionKYCRepository {
 
 class JdbcTransactionKYCRepository(private val db: Database) : TransactionKYCRepository {
   init {
-    transaction(db) { SchemaUtils.create(TransactionKYCs) }
+    transaction(db) {
+      SchemaUtils.create(TransactionKYCs)
+      val missingColumnsStatements = SchemaUtils.addMissingColumnsStatements(TransactionKYCs)
```

Comment:
Does this happen when we run the newer version of AP on the previous version of the DB?

Reply:
Yeah, that's right.